onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-18.10-20.101.sql

```diff
@@ -114,115 +114,6 @@ GO
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 2/17/2018 Jones ga
- * This script creates the ONPRC_EHR.IBC_Numberss Dataset which is populated by the ETL Process
- *
- */
-
-CREATE TABLE [onprc_ehr].[PRIME_VIEW_IBC_NUMBERS](
-    [rowid] [int] IDENTITY(1,1) NOT NULL,
-    [Animal_Group] [varchar](255) NOT NULL,
-    [IBC_Registration_Number] [varchar](255) NULL,
-    [createdby] [int] NULL,
-    [created] [datetime] NULL,
-    [modifiedby] [int] NULL,
-    [modified] [datetime] NULL
-    ) ON [PRIMARY]
-    GO
-
-/* Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 2/17/2018 Jones ga
- * This script creates the ONPRC_EHR.PRIME_VIEW_NON_SURGICAL_PROCS Dataset which is populated by the ETL Process
- *
- */
-
-CREATE TABLE [onprc_ehr].[PRIME_VIEW_NON_SURGICAL_PROCS](
-    [rowid] [int] IDENTITY(1,1) NOT NULL,
-    [Animal_Group] [varchar](255) NOT NULL,
-    [NS_Procedure_Name] [varchar](255) NULL,
-    [Standard_Procedure] [int] NULL,
-    [Iterations] [int] NULL,
-    [Deviation] [int] NULL,
-    [Deviation_Description] [varchar](255) NULL,
-    [Recovery_Days] [int] NULL,
-    [createdby] [int] NULL,
-    [created] [datetime] NULL,
-    [modifiedby] [int] NULL,
-    [modified] [datetime] NULL
-    ) ON [PRIMARY]
-    GO
-
-/* Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 2/17/2018 Jones ga
- * This script creates the ONPRC_EHR.PRIME_VIEW_PROTOCOL_Processing Dataset which is populated by the ETL Process
- *
- */
 
 CREATE TABLE [onprc_ehr].[PRIME_VIEW_PROTOCOL_Processing](
```

Comment:
This one should get deleted too

Reply:
OK will remove and reload

Reply:
Copncentrating on clean up of onprc_ehr-18..10-20.101 and removing all reference to eiACUC processsing, this was 3 year old code will be updating the build in the next 30 minutes

Reply:
Push completed My review shows all references to eIACUC processing has been removed